src/makeInfinityAdder.js

```diff
@@ -4,7 +4,21 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sumOfNumbers = 0;
+
+  return function adder(number = NaN) {
```

Comment:
The default parameter `number = NaN` is used to handle cases where no argument is passed. This is a good approach to ensure that the function can handle such cases gracefully.